### ISSUE TITLE
Add a Font Family

### ISSUE THREAD

**Issue by JavaScriptErika** [CascadingCoders/LandingPage#5]
Add a font family or 2 for our page from Google Fonts! Please add this in `index.html` and then in `src/styles/index.css`

**Comment by JavaScriptErika** [comment]
You may claim this issue, but please wait, as we are pending design. I will tag you when we're ready to proceed.

**Comment by Swayamsvk** [comment]
Can I work on this?

**Comment by JavaScriptErika** [comment]
@Swayamsvk yes! We are waiting on design - I will notify you when you can work on it. Thank you - it should be soon!

**Comment by meghnaaallam** [comment]
Can I work on this ?

**Comment by JavaScriptErika** [comment]
Hi @meghnareddy1999 - thank you for your interest! Swayamsvk has claimed this issue already. We have a couple of other open issues right now, and plan to have more in the future as the landing page progresses if you want to keep your eye out. Thanks!

**Comment by JavaScriptErika** [comment]
@Swayamsvk this task is ready for you to work on! Please take a look at issue #6  to get an idea of the wireframe and add in a google Font Family or two of your choice.

**Comment by JavaScriptErika** [comment]
Hi @Swayamsvk!

What is your current status? Thanks!
